aggregator/hitbtc.go

```diff
@@ -0,0 +1,46 @@
+package aggregator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+const (
+	hitbtcPriceUrl = "https://api.hitbtc.com/api/3/public/ticker/%s%s"
+)
+
+type hitbtc struct {
+	ResponseGetter
+}
+
+type hitbtcPriceRequest struct {
+	Price string `json:"last"`
+}
+
+// FetchPrice will fetch the price using the http client
+func (b *hitbtc) FetchPrice(ctx context.Context, base, quote string) (float64, error) {
+	if strings.Contains(quote, QuoteUSDFiat) {
```

Comment:
does it need a strings.Contains(**strings.ToUpper(quote)**, QuoteUSDFiat)?

Reply:
not really, it is already received as upper case. It will be fixed in the next PR where this check is extracted in a baseFetcher component, in order to remove duplicate code.